### PR TITLE
[-] BO : #PSCSX-5936 Fix e-mail preview

### PR DESCRIPTION
**Pull request by maximebiloe**
http://forge.prestashop.com/browse/PSCSX-5936

### CODE REVIEW

File: admin-dev/themes/default/template/controllers/translations/helpers/view/translation_mails.tpl [hunk 1]
```diff
@@ -123,7 +123,25 @@
...
+									url:'ajax.php',
+									type: 'POST',
+									async: false,
+									dataType: 'html',
+									data: {
+										getEmailHTML : true,
+										email : src
+									},
+									success: function(result)
+									{console.log(result);
```
Comment (tchauviere) [inline admin-dev/themes/default/template/controllers/translations/helpers/view/translation_mails.tpl]: A little console.log() has been forgotten :dart:

**Comment by tchauviere** [comment]
:+1:

**Comment by gRoussac** [comment]
Please make it again on 1.6.1.x branch + PSR2

**Comment by maximebiloe** [comment]
Just made a new PR : #4029

**Comment by gRoussac** [comment]
:lollipop:

**Pull request closed without merging**